README.md

```diff
@@ -10,24 +10,15 @@ able to run anywhere that Kubernetes can run.
 ##### Frontend
 
 ```
-$ cd Frontend
-$ yarn
-$ yarn build
-
-#### Node Server
-cd server
-yarn
+npm run-script build
```

Comment:
can we use `yarn` rather than `npm`?

Reply:
why? I am having a hard time with yarn permissions on openshift...

Reply:
just to be consistent, no big deal.